### ISSUE TITLE
Automate deploy on new commits in master

### ISSUE THREAD

**Issue by sindresorhus** [gulpjs/plugins#150]
We can use Travis to deploy. We do that on yeoman.io and todomvc.com.

https://github.com/yeoman/yeoman.io/issues/344

**Comment by contra** [comment]
@arthurvr Any updates?

**Comment by arthurvr** [comment]
>  Any updates?

I ran into issues with the build and didn't work on this more. I guess it's fixed these days so let me try again.

**Comment by phated** [comment]
Closing this because the plugin page has been rewritten and integrated with our main website: https://github.com/gulpjs/gulpjs.github.io
